main.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace labs;
+
+use labs\Department;
+use labs\Employee;
+
+include 'Employee.php';
+include 'Department.php';
+require 'vendor/autoload.php';
+
+use Symfony\Component\Validator\Validation;
+
+$validator = Validation::createValidatorBuilder()
+    ->addMethodMapping('loadValidatorMetadata')
+    ->getValidator();
+
+
+$em1 = new Employee(1, "Stepan", 700, "2019-11-12");
+$em2 = new Employee(2, "Andrey", 400, "2023-11-12");
+$em3 = new Employee(3, "Kirill", 300, "2021-11-12");
+$em4 = new Employee(4, "Sergey", 600, "2019-11-12");
+$em5 = new Employee(5, "Anton", 400, "2017-11-12");
+$em6 = new Employee(7, "Vladimir", 300, "2012-11-12");
+$em7 = new Employee(8, "Danil", 200, "2021-11-12");
+$em8 = new Employee(9, "Nikita", 100, "2020-11-12");
+
+//Validation test
+$em9 = new Employee(-1, "Artem", 100, "2022-05-11");
+$em10 = new Employee(11, "Al", 100, "2021-11-09");
+$em11 = new Employee(10, "Igor", -100, "2018-04-15");
+$em12 = new Employee(12, "Pavel", 100, "20-02");
+
+$em_array = array($em1, $em2, $em3, $em4, $em5, $em6, $em7, $em8, $em9, $em10, $em11, $em12);
```

Comment:
Для создания массива лучше использовать []
```suggestion
$emArray = [$em1, $em2, $em3, $em4, $em5, $em6, $em7, $em8, $em9, $em10, $em11, $em12];
```